docs/update-covid-local-restriction-data.md

```diff
@@ -0,0 +1,67 @@
+# Update COVID Local Restriction Data
+
+### Very brief description of how the checker works
+
+- The postcode checker makes calls to Mapit to get information about the submitted postcode.
+- The Mapit response contains `areas` that contain `GSS` codes for the postcode.
+- We select the GSS code for the lower tier local authority of the areas returned.
+- We then use the GSS code to query our local restriction YAML [data](https://github.com/alphagov/collections/blob/master/lib/local_restrictions/local-restrictions.yaml) to figure out which restriction the `GSS` code is under.
+
+### How to update the local restriction data
+
+[Here is an example PR!](https://github.com/alphagov/collections/pull/2009/files)
+
+In order to update the local restriction data, you will need to be given the `GSS` codes and new alert levels of the places for which restrictions are changing.
+
+You will also be told the time and date at which the alert levels will change.
+
+We display both the current and upcoming restrictions to a user IF we have upcoming restrictions for their postcode.
+
+![Image of results page with changing alert level](docs/alert-level-changing.png)
+
+Otherwise we just display the current restrictions. This is determined by the `start_date` and `start_time`.
+
+![Image of results page without changing alert level](docs/alert-level-no-upcoming-changes.png)
+
+Example of somewhere that just has current restrictions:
+
+```
+E08000012:
+  name: Liverpool City Council
```

Comment:
One thing we should mention is that we use Mapit to look up the "name" we enter here.

Reply:
thanks, have added :)